compiler/src/stackAlloc.ml

```diff
@@ -205,7 +205,21 @@ let memory_analysis pp_err ~debug up =
   
   (* remove unused result *)
   let tokeep = RemoveUnusedResults.analyse fds in
-  let tokeep fn = tokeep fn in
+  (* FIXME: the code is duplicated between here and compiler.v, this is horrible *)
```

Comment:
FIXME remains. This might be turned into a TODO, with a less scary comment.

Reply:
Done.